README.md

```diff
@@ -1,70 +1,89 @@
-# Getting Started with Create React App
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-## Available Scripts
+# ANY COUNTRY INFO
 
-In the project directory, you can run:
+Single page application giving general informations about  any country in the world.
 
-### `npm start`
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+## Built With
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+- HTML3
+- CSS5
+- React with JSX
+- REDUX
+- API
+- Testing
+
+## Live Demo 
+
+[Live Demo Link](https://pamphilemkp-countries-info.netlify.app/)
+
+## Live video
+
+[Video](https://www.loom.com/share/dc14a5e5b4b540d2a347d19fef80c975)
```

Comment:
- [ ] The video should contain a highlight about a piece of the code the student finds interesting or is proud of making, kindly consider showing some code on your video presentation not only the app